### PR TITLE
adding recipe for migrating from Onenote

### PR DESCRIPTION
**Pull request by nixsee**
I have made a script for exporting from Onenote to markdown [here](https://github.com/nixsee/ConvertOneNote2MarkDown)

I'm happy to help integrate it into your recipes for others to use

### CODE REVIEW

File: docs/roadmap.md [hunk 1]
```diff
@@ -70,8 +70,9 @@ If a roadmap item is a stub, **consider** opening a [GitHub issue](https://githu
   - Discussion: [foam#55](https://github.com/foambubble/foam/issues/55)
 - [[migrating-from-obsidian]]
   - Discussion: [foam#46](https://github.com/foambubble/foam/issues/46)
+- [[Migrating from OneNote (stub)]]
+  Discussion: [foam#151](https://github.com/foambubble/foam/issues/151)
```
Comment (jsjoeio) [inline docs/roadmap.md:74]:
```suggestion
  - Discussion: [foam#151](https://github.com/foambubble/foam/issues/151)
```

Comment (jsjoeio) [reply docs/roadmap.md:74]: I think this is missing a `-` to make it a sub-bullet. Or at least now it matches the others

**Comment by jsjoeio** [comment]
@nixsee wow, you're quick! Thanks for adding this 😄 One nitpick/suggestion added above. Feel free to accept it and then ping me after and we can approve!

**Comment by jsjoeio** [comment]
Someone else from @foambubble/contributors will merge this and then we'll add you to the contributors table!

**Comment by nixsee** [comment]
Thanks for being so gentle for my first time! Looking forward to contributing how I'm able!

**Pull request merged**